src/modules/compliance/src/so/ComplianceModule.c

```diff
@@ -5,14 +5,26 @@
 #include "ComplianceInterface.h"
 #include <stddef.h>
 
+static OsConfigLogHandle gLog = NULL;
+static const char* gLogFile = "/var/log/osconfig_compliance.log";
+static const char* gRolledLogFile = "/var/log/osconfig_compliance.bak";
+
 void __attribute__((constructor)) InitModule(void)
 {
-    ComplianceInitialize(NULL);
+    gLog = OpenLog(gLogFile, gRolledLogFile);
+    if (NULL == gLog)
+    {
+        OsConfigLogError(NULL, "Failed to open log file");
+        return;
```

Comment:
What will happen if we return here, without ComplianceInitialize? Shouldn't that be an assert?